Services/CO.CDP.Tenant.WebApi.Tests/Api/TenantLookupTest.cs

```diff
@@ -82,7 +82,8 @@ private static TenantLookup GivenTenantLookup()
             {
                 Urn = "urn:fdc:gov.uk:2022:43af5a8bf4c0414bb341d4f1fa894302",
                 Name = "Alice Cooper",
-                Email = "person@example.com"
+                Email = "person@example.com",
+                Scopes = []
```

Comment:
Can we verify non empty scopes in one of the tests please?

Reply:
done.